windows-registry/Cargo.toml

```
@@ -0,0 +1,25 @@
+[package]
+name = "windows-registry"
+version = "0.1.0"
+edition = "2018"
+authors = ["Alex Crichton <alex@alexcrichton.com>"]
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/rust-lang/cc-rs"
+homepage = "https://github.com/rust-lang/cc-rs"
+documentation = "https://docs.rs/windows-registry"
+description = """
+A build-time dependency for Cargo build scripts to assist in invoking the native
+C compiler to compile native C code into a static archive to be linked into Rust
+code.
+"""
+keywords = ["build-dependencies"]
+readme = "../README.md"
```

Nit:

```suggestion
name = "windows-registry"
version = "0.1.0"
edition = "2018"
license = "MIT OR Apache-2.0"
repository = "https://github.com/rust-lang/cc-rs"
documentation = "https://docs.rs/windows-registry"
description = "Read MSVC versions from the registry and from COM interfaces"
keywords = ["build-dependencies"]
readme = "../README.md"
```

Also, the crate should probably have its own README if we're to treat it as a public crate.